00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
Also, ROCK_WEBAPP is a single path, and does not support having multiple paths at the moment. This could be added later, but may not even be needed.